hivemind/p2p/p2p_daemon.py

```diff
@@ -28,13 +28,11 @@ class P2PInterruptedError(Exception):
     pass
 
 
-@dataclass(frozen=False)
+@dataclass
 class P2PContext(object):
-    id: str
-    port: int
     handle_name: str
-    peer_id: PeerID = None
-    peer_addr: Multiaddr = None
+    client_id: PeerID = None
+    client_maddr: Multiaddr = None
```

Comment:
Naming in `P2PContext` was misleading: `id` and `port` denoted the server (local) credentials, `peer_id` and `peer_addr` denoted the client (remote) credentials.

This change makes naming more clear and removes the local credentials since they are already accessible as `P2P` class fields.